groups/reconcile.go

```diff
@@ -202,30 +202,66 @@ func readConfig(configFilePath string, confirmChanges bool) error {
 	return err
 }
 
-func readGroupsConfig(configFilePath string, groupsConfigFilePath string, config *GroupsConfig) error {
+func readGroupsConfig(configFilePath string, groupsFileName string, config *GroupsConfig) error {
```

Comment:
Capitalization nit
```suggestion
func readGroupsConfig(configFilePath string, groupsFilename string, config *GroupsConfig) error {
```

As with `readMultipleGroupsConfig` I think this should take an explicit dir instead of passing along `configFilePath` and deriving from it later